data/categories.txt

```diff
@@ -7793,6 +7793,47 @@ sw:Uwanja wa gofu|michezo
 fa:زمین گلف
 mr:गोल्फचे मैदान
 
+leisure-hackerspace
+en:4Hackerspace
+ar:هاكرزبيس
+be:Хакерская прастора
+bg:Hackerspace
+ca:Hackerspace
+cs:Hackerspace
```

Comment:
if there is no translation then its better to delete such a row;
`en:` version is used for search always AFAIK